resources/views/users/show.blade.php

```diff
@@ -55,20 +56,20 @@
                                         {{ $competition->id }}
                                     </a>
                                 </td>
-                                <td class="table__cell">
-                                    @if ($competition->reportback)
-                                        <a href="{{ $competition->reportback->admin_url }}">
-                                            {{ $competition->reportback->id }}
-                                        </a>
-                                    @else
-                                        N/A
-                                    @endif
-                                </td>
-                                <td class="table__cell">{{ $competition->reportback->reportback_items->total or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->quantity or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->updated_at or 'N/A'}}</td>
-                                <td class="table__cell">{{ $competition->reportback->flagged or 'N/A' }}</td>
-                                <td class="table__cell"><a href="{{ route('competitions.users.destroy', ['user' => $user->id, 'competition' =>$competition->id]) }}" class="button -danger">Remove</a></td>
+
+                                @if ($competition->reportback)
+                                    <td class="table__cell"><a href="{{ reportback_admin_url($competition->reportback->id) }}">{{ $competition->reportback->id }}</a></td>
+                                    <td class="table__cell">{{ $competition->reportback->reportback_items->total }}</td>
+                                    <td class="table__cell">{{ $competition->reportback->quantity }}</td>
+                                    <td class="table__cell">{{ format_timestamp_for_display($competition->reportback->updated_at) }}</td>
+                                @else
+                                    <td class="table__cell">N/A</td>
```

Comment:
It's also cause now I'm just appending the Reportback to the competition. If there isn't any, it's `null`. Wanted to go about using similar approaches w/ how RBs are appended to collection of users, single user, or competition.